### DISCUSSION TITLE
Add more useful "last saved" relative time stamps

### DISCUSSION THREAD

**Discussion by yandodov** [payloadcms/payload#2876] [Feature Requests & Ideas]
Currently, the "last saved" text only displays relative time in minutes. That's pretty useless once a few hours have passed, especially days:

![image](https://github.com/payloadcms/payload/assets/5570098/eb02f8ea-8e22-4814-bd3d-bed53497cbe2)

It'd be great if it works more traditionally and moves from minutes to hours, days, months, and years.

**Comment by DanRibbens** [comment]
I think this could be considered a regression, I'm pretty sure it was behaving the way you described. I remember trying to implement it this way when making the admin UI internationalized. If you're interested in making a contribution I can add some notes here on how to fix it.
